assessments/projects/rook/self-assessment.md

```diff
@@ -0,0 +1,374 @@
+# Self-assessment
```

Comment:
Please note the project name in the H1 heading, like # Rook Self-asssesment

Reply:
Fixed in this commit: [cfbb19f](https://github.com/dhauss/tag-security/commit/cfbb19f0ab420fbffd3366d72b2d3c037085bf8d)